stan/math/rev/mat.hpp

```diff
@@ -5,6 +5,7 @@
 #include <stan/math/rev/scal/meta/is_var.hpp>
 #include <stan/math/rev/scal/meta/partials_type.hpp>
 
+#include <stan/math/rev/mat/vectorize/apply_scalar_unary.hpp>
```

Comment:
this include should go at the end, unless there's some wicked order dependency.... if there is, then we should figure out why.